sound/soc/sof/ipc4-topology.c

```diff
@@ -418,6 +404,23 @@ static int sof_ipc4_widget_setup_pcm(struct snd_sof_widget *swidget)
 	return ret;
 }
 
+static void sof_ipc4_widget_free_comp_pcm(struct snd_sof_widget *swidget)
```

Comment:
@ranj063 @bardliao for my education is 'aif' a synonym for 'pcm'?

Reply:
yes